qemu/tests/cfg/uefishell.cfg

```diff
@@ -18,6 +18,12 @@
             command_connect = "connect -r"
             command_oprom = "dh -d -v -p SimpleNetwork"
             check_result_oprom = "Controller Name.*:.*iPXE"
+        - check_memory_map:
+            test_scenarios = "connect memmap"
+            command_connect = "connect -r"
+            command_memmap = "memmap"
+            check_rt_code = "^(?!RT_Code\s+\w+-\w+\s+\w+\s+\w+\s+RT_Code).*"
+            handle_memmap = "handle_memory_map"
```

Comment:
what about 'handle_memmap'->'memmap_output_handler', 'check_rt_code' -> 'memmap_rt_code'

Reply:
agree with you, thanks